browser/src/Services/Browser/BrowserView.tsx

```diff
@@ -157,9 +158,21 @@ export class BrowserView extends React.PureComponent<IBrowserViewProps, IBrowser
             scrollLeft,
             scrollRight,
         ])
+
         this._initializeElement(this._elem)
     }
 
+    public toggleInputStatus = () => {
+        this._webviewElement.executeJavaScript(
```

Comment:
Ideally we could 'push' this information up, instead of having to check everytime a key is pressed (which will hurt the responsiveness of key presses). In #2239 I set up this 'push' pipeline where the `webview` tag will let us know whenever the focused tag changes - that way we don't have to do stuff on keypress.